sql/src/test/java/org/apache/druid/sql/calcite/DrillWindowQueryTest.java

```diff
@@ -377,12 +381,10 @@ public void verify(String sql, QueryResults queryResults)
       }
     }
 
-    private boolean isOrdered(String sql)
+    private boolean isOrdered(QueryResults queryResults)
     {
-      // FIXME: SqlToRelConverter.isOrdered(null) would be better
-      sql = sql.toLowerCase(Locale.ENGLISH).replace('\n', ' ');
-      sql = sql.substring(sql.lastIndexOf(')'));
-      return sql.contains("order");
+      SqlNode sqlNode = ((PlannerCaptureHook) queryResults.capture).getSqlNode();
```

Comment:
```suggestion
      SqlNode sqlNode = queryResults.capture.getSqlNode();
```

this can be done in any later change, no need to run CI for this!

Reply:
sure; I think my IDE will remove this cast right away when I next change that file